modules/sharethroughBidAdapter.js

```diff
@@ -129,37 +129,39 @@ export const sharethroughAdapterSpec = {
             [w, h] = videoRequest.playerSize[0];
           }
 
-          const getVideoPlacementValue = (vidReq) => {
-            if (vidReq.plcmt) {
-              return vidReq.placement;
-            } else {
-              return vidReq.context === 'instream' ? 1 : +deepAccess(vidReq, 'placement', 4);
+          /**
+           * Applies a specified property to an impression object if it is present in the video request
+           * @param {string} prop A property to apply to the impression object
+           * @param {object} vidReq A video request object from which to extract the property
+           * @param {object} imp A video impression object to which to apply the property
+           */
+          const applyVideoProperty = (prop, vidReq, imp) => {
+            const propTakesArrayValues = ['api', 'mimes', 'playbackmethod', 'protocols'].includes(prop);
```

Comment:
Nitpick: I had to reread it a couple of times to understand the meaning. Would you be ok with:

```suggestion
            const propIsTypeArray = ['api', 'mimes', 'playbackmethod', 'protocols'].includes(prop);
```

Reply:
Updated.